invoicex/main.py

```diff
@@ -5,8 +5,9 @@
 import os
 import time
 
-import reader
+import invoicex.reader.github as github
```

Comment:
```suggestion
from invoicex.reader import ComposeReader
```

the idea is to hide all the complexity and use ComposeReader as the main interface for the reading.


ComposeReader will receive the parameters about which datasets/data-sources will be used (for now, we just have github and ttrack)

Reply:
please check this comment